### PR TITLE
Add Roborock Q Revo

### PR DESCRIPTION
**Pull request by pablolop002**
Draft as I have to test it properly

### CODE REVIEW

File: lib/modules/robotcleaner/devices/roborock.vacuum.a75.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+  /*----------========== DEVICE INFO ==========----------*/
+
+  getDeviceName() {
+    return "Roborock S8 Pro Ultra";
+  }
+
+  getMiotSpecUrl() {
+    return "https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:vacuum:0000A006:roborock-a70:1";
```
Comment (merdok) [inline lib/modules/robotcleaner/devices/roborock.vacuum.a75.js:21]: does not look correct

File: lib/modules/robotcleaner/devices/roborock.vacuum.a75.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+class RoborockVacuumA75 extends RobotCleanerDevice {
+  constructor(miotDevice, name, logger) {
+    super(miotDevice, name, logger);
+  }
+
+
+  /*----------========== DEVICE INFO ==========----------*/
+
+  getDeviceName() {
+    return "Roborock S8 Pro Ultra";
```
Comment (merdok) [inline lib/modules/robotcleaner/devices/roborock.vacuum.a75.js:17]: also does not look correct

**Comment by merdok** [comment]
I found two places which are probably copy paste issues, could you also adjust those? Other then that, after you test it we can merge it.

**Comment by pablolop002** [comment]
> I found two places which are probably copy paste issues, could you also adjust those? Other than that, after you test it we can merge it.

Maybe. I tried to generate the file with the option given in the Homebridge UI, but the file was generated without headers.

**Comment by merdok** [comment]
Added in version 1.7.1

**Pull request closed without merging**